de.dlr.sc.virsat.model.edit.test/src/de/dlr/sc/virsat/model/concept/types/property/BeanPropertyBooleanTest.java

```diff
@@ -97,5 +97,12 @@ public void testEqualsObject() {
 		
 		beanProperty2.setTypeInstance(vpi);
 		assertEquals("Beans are identical", beanProperty2, beanProperty);
+		

```

Comment:
Empty line nod needed and can be removed

Reply:
Removed it.